src/Widgets/FormHiddenFieldWidget/FormHiddenFieldWidgetEditingConfig.js

```diff
@@ -19,7 +19,10 @@ Scrivito.provideEditingConfig("FormHiddenFieldWidget", {
   },
   validations: [customFieldNameValidation],
   titleForContent: (widget) =>
-    [widget.get("customFieldName"), truncate(widget.get("hiddenValue"))]
-      .filter((e) => e)
-      .join(": "),
+    truncate(
+      `Hidden: ${[widget.get("customFieldName"), widget.get("hiddenValue")]
+        .filter((e) => e)
+        .join(" - ")}`,
+      { length: 36 }
```

Comment:
* [ ] The UI already trims titles where needed. In places where there's enough room, the full title is better.